DevEdu.API/DevEdu.API/Controllers/TaskController.cs

```diff
@@ -70,29 +80,53 @@ public string DeleteTagFromTask(int taskId, int tagId)
 
         // api/task/{taskId}/student/{studentId}
         [HttpPost("{taskId}/student/{studentId}")]
-        public string AddStudentAnswerOnTask(int taskId, int studentId, string taskAnswer)  // to inputModel
+        public string AddStudentAnswerOnTask(int taskId, int studentId, [FromBody] StudentAnswerOnTaskInputModel studentResponse)

```

Comment:
please rename "studentResponse" into "inputModel"
otherwise it might be confusing)